BootloaderCommonPkg/Include/Guid/LoaderPlatformInfoGuid.h

```diff
@@ -24,23 +24,45 @@ extern EFI_GUID gLoaderPlatformInfoGuid;
 
 #pragma pack(1)
 
+//Definition for LOADER_PLATFORM_INFO.LdrFeatures
```

Comment:
Same definition here - BootloaderCommonLib.h. Need to consolidate.

Reply:
Yeah, one set of definitions are used in Stage1 and another set is used a part HOB data. 
I would try to consolidate and submit another patch.